lib/crystalball/rspec/runner/configuration.rb

```diff
@@ -1,18 +1,16 @@
 # frozen_string_literal: true
 
-require 'crystalball/simple_predictor'
-
 module Crystalball
   module RSpec
     class Runner
       # Class for storing local runner configuration
       class Configuration
         def initialize(config = {})
           @values = {
-            'map_path' => 'tmp/execution_maps',
+            'execution_map_path' => 'tmp/execution_map.yml',
             'map_expiration_period' => 86_400,
             'repo_path' => Dir.pwd,
-            'predictor_class_name' => 'Crystalball::SimplePredictor',
+            'prediction_builder_class_name' => nil,
```

Comment:
~simple predictor~simple prediction builder is actually a good idea to provide something out of the box